build/source/engine/canopySnow.f90

```diff
@@ -123,10 +132,14 @@ subroutine canopySnow(&
 
  ! model prognostic variables (input/output)
  scalarCanopyIce           => prog_data%var(iLookPROG%scalarCanopyIce)%dat(1),             & ! intent(inout): [dp] mass of ice on the vegetation canopy (kg m-2)
+ scalarCanairTemp          => prog_data%var(iLookPROG%scalarCanairTemp)%dat(1),             & ! intent(input): [dp] temperature of the canopy air space (k)
```

Comment:
align `&`

Reply:
Use `intent(in)` rather than `intent(input)`

Reply:
Done, and moved to other `intent(in)` section